src/MISC/fix_deposit.cpp

```diff
@@ -766,6 +776,13 @@ void FixDeposit::options(int narg, char **arg)
       vzlo = force->numeric(FLERR,arg[iarg+1]);
       vzhi = force->numeric(FLERR,arg[iarg+2]);
       iarg += 3;
+    } else if (strcmp(arg[iarg],"rotation") == 0) {
```

Comment:
wouldn't it be better to call this keyword "orientation" instead of "rotation"?

Reply:
Hi, akohlmey! Sure, many thanks for your help! I have changed all the lines based on your comments!!